Flow.Launcher.Test/HttpTest.cs

```diff
@@ -0,0 +1,33 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Flow.Launcher.Infrastructure.UserSettings;
+using Flow.Launcher.Infrastructure.Http;
+
+namespace Flow.Launcher.Test
+{
+    [TestFixture]
+    class HttpTest
+    {
+        [Test]
+        public void TestSettingUpdate()
```

Comment:
can you give this a bit more description please, Given-When-Then

Reply:
ok, I will add comment.

Reply:
hey if it's ok, i will update it for you, sorry i need to be afk soon and i want to get this release ready, so when you are free have a look at 1.7.2 and approve if all good, i will release it later

Reply:
Yeah sure, if you would like to do it will be great. I just have dinner so haven't finish that part.

Reply:
I will approve 1.7.2 once this has been merged. 🥰